autopush/router/simple.py

```diff
@@ -70,6 +78,12 @@ def route_notification(self, notification, uaid_data):
         # Determine if they're connected at the moment
         node_id = uaid_data.get("node_id")
         uaid = uaid_data["uaid"]
+        self.udp = None
+        try:
+            self.udp = json.loads(uaid_data["udp"])
```

Comment:
Also, uaid_data is a JSON dict out of DynamoDB. DynamoDB items can store structured nested dicts, so in theory we don't need to JSON encode it on the way in/out like this.